frontend/review/src/components/Announcement.js

```diff
@@ -0,0 +1,197 @@
+import React, { useEffect, useRef, useState } from "react"
+
```

Comment:
Does `pcx-shared-components` not work on pcr?

Reply:
I assumed that from the [README](https://github.com/pennlabs/penn-courses/tree/master/frontend). Let me trying moving that into shared components instead, but I'm not sure how using TypeScript in PCR will play out here!